### ISSUE TITLE
RuntimeError when extracting text from EPUB

### ISSUE THREAD

**Issue by mush42** [pymupdf/PyMuPDF#540]
## Describe the bug
When openning an EPUB document, PyMuPDF raises RuntimeError('Not a PDF') when attempting to extract text using any of the provided text extraction methods: document[n].getText and document.getPageText.

## To Reproduce
```python
import fitz

doc = fitz.open(".\\foo.epub")
doc[1].getText()
```

PyMuPDF raises RuntimeError with the following traceback:

```python
mupdf: not a PDF
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\DELL\bookworm\.venv64\lib\site-packages\fitz\utils.py", line 468, in getText
    tp = page.getTextPage(flags)  # TextPage with or without images
  File "C:\Users\DELL\bookworm\.venv64\lib\site-packages\fitz\fitz.py", line 3716, in getTextPage
    self.setRotation(0)
  File "C:\Users\DELL\bookworm\.venv64\lib\site-packages\fitz\fitz.py", line 4274, in setRotation
    return _fitz.Page_setRotation(self, rotation)
RuntimeError: not a PDF
```

## Expected behavior
`doc[n].getText()` should return the page's text.

## Your configuration
 - Operating system: Windows 10Ver1903 (64-bit)
 - Python version: Python 3.8.3 (64-bit)
 - PyMuPDF version: latest version from PyPI (PyMuPDF v1.17.2).
 
 ```python
 >>> print(fitz.version)
 ('1.17.2', '1.17.0', '20200620070022')
 ```

**Comment by JorjMcKie** [comment]
Thanks for submitting this.
Error identified. Any urgency - please drop a note if you would like a pre-release wheel.

**Comment by mush42** [comment]
No thanks. I'll wait for the official release.

**Comment by JorjMcKie** [comment]
fixed in version 1.17.3 just publish
